src/turbofish.rs

```diff
@@ -43,32 +59,25 @@ impl<'de> Deserialize<'de> for TurboFish {
 }
 
 fn parse(param: &str) -> Option<TurboFish> {
-    match param.as_bytes().get(..3)? {
-        b"::<" => {
-            let mid = param[3..].strip_suffix('>')?;
-            Some(TurboFish::new(mid.to_owned()))
+    match param.as_bytes() {
+        b"::()" | b"::<()>" => return Some(TurboFish::new(Fish::Unit)),
```

Comment:
I don't understand. What is `::()` for?

Reply:
```rust
let _ = Fn::() -> ()::call;
```

is valid Rust syntax. Thus forward and backward turbo-unit :)

Reply:
We could consider that things such as `impl/dyn Fn ::()` yield a nice turbo-_jellyfish_, maybe? 🙃

Reply:
Alright, I'm convinced. However @mattfbacon I'm not super happy about all the unrelated changes. Could you just add static `/::()` and `/()::` routes with their own template? Since this is basically an easter egg that you can only see by knowing the URL, maybe we'd want to adjust its look more too, or I'll look at a nice way to generalize it myself.

Re. that `f.alternate()` hack, I find it very fun but could you move that to a separate PR with a small comment like `alternate formatting reverses the turbofish`?